### PR TITLE
Game styling + additional functionalities

### PR DESCRIPTION
**Pull request by kndshein**

### CODE REVIEW

File: src/components/LetterSelection/index.jsx [hunk 1]
```diff
@@ -77,51 +97,96 @@ const reducer = (state, action) => {
...
+                payload: {
+                  clickedLetters: letterState.clickedLetters,
+                  letterDispatch: dispatch,
+                  setCorrectInLetters: types.setCorrectInLetters,
+                },
               })
             }
           >
-            {selectableLetter}
+            <GrReturn />
```
Comment (daleinen7) [inline src/components/LetterSelection/index.jsx:186]: More of a style choice than a coding problem, but I think this should be above the backspace. I see the logic for putting it below, but I think it feels better above.

File: src/components/Sentence/index.jsx [hunk 2]
```diff
@@ -1,24 +1,8 @@
...
+export default function Sentence({ sentence }) {
   return (
     <>
-      <h2
-        style={{ visibility: wordTimer ? "hidden" : "" }}
-        className={wordTimer ? "hidden" : ""}
-      >
-        {word}
-      </h2>
       <h2>{sentence}</h2>
```
Comment (daleinen7) [inline src/components/Sentence/index.jsx:6]: This should probably be a p tag.

**Review by daleinen7 — APPROVED** [review]
Looks really good. I left notes, but I'll let you decide whether you really want to address them or not. I could live with it as is if I needed to.

File: src/styles/_variables.scss [hunk 3]
```diff
@@ -1,5 +1,44 @@
+@import url("https://fonts.googleapis.com/css2?family=Quicksand:wght@500&display=swap");
```
[the remark this answers is not in the document]
Comment (kndshein) [reply src/styles/_variables.scss:1]: That is perfect.

File: src/components/FlashWord/index.jsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    } else {
+      setAppear(false);
+    }
+  }, [word]);
+
+  return (
+    <>
+      <h2
+        style={{ visibility: appear ? "" : "hidden" }}
+        className={appear ? "" : "hidden"}
```
[the remark this answers is not in the document]
Comment (kndshein) [reply src/components/FlashWord/index.jsx:22]: It won't need both lines, I'm still messing with the hidden functionality. The `style` shall be removed soon

File: src/components/LetterSelection/index.jsx (hunk 1 above)
Comment (kndshein) [reply src/components/LetterSelection/index.jsx:186]: What should be above the backspace? The submit button?

Comment (daleinen7) [reply src/components/LetterSelection/index.jsx:186]: Correct. Is this not the right spot that is. I just assumed it was the Green Return symbol.

Comment (kndshein) [reply src/components/LetterSelection/index.jsx:186]: I disagree that the backspace button should be below the return symbol. (i.e. from physical keyboards to on-screen touch ones, backspace is always above the return key)

Comment (daleinen7) [reply src/components/LetterSelection/index.jsx:186]: We'll I guess duel to the death is the only way to resolve this empasse.

**Pull request merged**